### DISCUSSION TITLE
Serif typeface used for crop dimensions

### DISCUSSION THREAD

**Discussion by alexjcollins** [craftcms/cms#8775] [Ideas]
### Description
Literally the tiniest of things, but would be nice to make uniform with the rest of the UI.

<img width="628" alt="Screenshot 2020-02-04 at 08 18 10" src="https://user-images.githubusercontent.com/12031196/73824355-eb624d80-47f1-11ea-9012-91e607e54558.png">

### Steps to reproduce

1. Crop an image asset

### Additional info

- Craft version: 3.4.3

**Comment by andris-sevcenko** [comment]
@alexjcollins yeah, I must have missed that, thanks for noticing!
